flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/IncrementalAggregateTimeWindowFunction.scala

```diff
@@ -55,10 +47,10 @@ class IncrementalAggregateTimeWindowFunction(
   }
 
   override def apply(
-    key: Tuple,
-    window: TimeWindow,
-    records: Iterable[Row],
-    out: Collector[Row]): Unit = {
+      key: Tuple,
```

Comment:
If we omit the grouping keys in the  `AggregateAggFunction` we can get them from the `Tuple`